### PR TITLE
behaviortree_cpp_v3: 3.0.8-1 in 'dashing/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by facontidavide**
Increasing version of package(s) in repository `behaviortree_cpp_v3` to `3.0.8-1`:

- upstream repository: https://github.com/BehaviorTree/BehaviorTree.CPP.git
- release repository: https://github.com/BehaviorTree/behaviortree_cpp_v3-release.git
- distro file: `dashing/distribution.yaml`
- bloom version: `0.8.0`
- previous version for package: `null`

### CODE REVIEW

**Comment by dirk-thomas** [comment]
@facontidavide This release is failing to build: http://build.ros2.org/view/Dbin_uB64/job/Dbin_uB64__behaviortree_cpp_v3__ubuntu_bionic_amd64__binary/5/

File: dashing/distribution.yaml [hunk 1]
```diff
@@ -160,6 +160,17 @@ repositories:
...
+  behaviortree_cpp_v3:
+    release:
+      tags:
+        release: release/dashing/{package}/{version}
+      url: https://github.com/BehaviorTree/behaviortree_cpp_v3-release.git
+      version: 3.0.8-1
+    source:
+      type: git
+      url: https://github.com/BehaviorTree/BehaviorTree.CPP.git
+      version: master
```
Comment (dirk-thomas) [inline dashing/distribution.yaml]: @facontidavide This branch name seems to be wrong. Should this be `ros2-dev` instead?

There seem to be numerous differences between the `master` and `ros2-dev` branch can you please clarify which branch you indent to use for ROS 2 development / releases?

Comment (dirk-thomas) [reply dashing/distribution.yaml]: ros/rosdistro#21342 switches the devel job to the suggested branch `ros2-dev`.

Comment (facontidavide) [reply dashing/distribution.yaml]: thanks. I messed up!

**Pull request merged**